src/resources/elements/primeDesignSystem/pbutton/pbutton.ts

```diff
@@ -19,5 +19,7 @@ export type ButtonType = "primary" | "secondary" | "tertiary";
 export class PButton {
   @bindable.string type: ButtonType;
   @bindable.booleanAttr disabled = false;
+  @bindable.booleanAttr doNotAnimate = false;
```

Comment:
can you please confirm whether secondary and tertiary are supposed to have animation?  They don't have it in your screenshots, but you seem to have been coded in the CSS to assume they do.

Be sure to to document the use cases in the viewmodel

Reply:
I've changed the `no-animation` to effct only the primary color transition. It currently means that there is no visable "hover" state for primary buttons if `no-animation` is selected.

`no-animation` have no effect over "secondary" and "tertiary".